src/remoteplayer.h

```diff
@@ -32,6 +32,16 @@ enum RemotePlayerChatResult
 	RPLAYER_CHATRESULT_KICK,
 };
 
+struct CloudSettings
```

Comment:
I'd probably change it to CloudParameters, though – the `Clouds` class already has a `readSettings` method, which is something different.